### PR TITLE
solution

### PR DESCRIPTION
**Pull request by YevhenOstrovskiy**
[DEMO LINK](https://YevhenOstrovskiy.github.io/react_people-table-basics/)

### CODE REVIEW

**Review by OleksandrRezanov — APPROVED** [review]
Good job!

File: src/components/PeopleList/PeopleList.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+export const PeopleList: React.FC = () => {
+  const { peoples, errorMessage } = useContext(PeopleContext);
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      { errorMessage === '' && (
```
Comment (lerastarynets) [inline src/components/PeopleList/PeopleList.tsx:13]:
```suggestion
      { !errorMessage && (
```

File: src/components/PeopleList/PeopleList.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          </thead>
+
+          <tbody>
+            {peoples.map((person) => (
+              <PersonItem person={person} key={person.slug} />
+            ))}
+          </tbody>
+        </>
+      )}
+      { peoples.length === 0 && errorMessage === '' && (
```
Comment (lerastarynets) [inline src/components/PeopleList/PeopleList.tsx:33]:
```suggestion
      { !peoples.length && !errorMessage && (
```
pls see [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)

**Review by lerastarynets — APPROVED** [review]
Great job overall! Left a few comments to make your code more advanced)

**Pull request closed without merging**